boards/nucleo144-f446/include/periph_conf.h

```diff
@@ -29,6 +29,7 @@ extern "C" {
  * @name    Clock system configuration
  * @{
  */
+#define P                       (4U)
```

Comment:
its the _PLL Output divisor_  see [here](https://github.com/RIOT-OS/RIOT/blob/master/cpu/stm32f4/stmclk.c#L50) and needs to be set to `4` to work with the cpu frequency here.